### PR TITLE
Fix for esbuild.github.io

### PR DESCRIPTION
**Pull request by bershanskiy**
Temporary fixes for gradient backgrounds since Dark Reader does not support resolving variables yet.

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -18094,6 +18103,12 @@ body,
 .topbar .icon-site-switcher-bubble {
     background-repeat: no-repeat !important;
 }
+div.h100.bar-lg.p-bs-wrapper:has(a[href="/questions"]) {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:18106]: FYI don't do multiple sites fixes in a PR, unless indicated in the commit title.

Comment (bershanskiy) [reply src/config/dynamic-theme-fixes.config:18106]: Yeah, the PR title was accidental (I created a PR and then remembered that I saw a very similar bug elsewhere). Also, a proper fix would be to implement support for lowering expressions with `background` set to `linear-gradient` dependent on some `var`.

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:18106]: Unfortunately that's not possible. The browser doesn't show a `var(...)` value for the `background` property. The browsers itself doesn't know how to resolve it at a CSS level.

Comment (bershanskiy) [reply src/config/dynamic-theme-fixes.config:18106]: stackoverflow.com has the following CSS (for example):
```css
background: linear-gradient(0deg, var(--orange-100) 30%, var(--white))
```
So we might want to lower that and substitute our vars in place of these?

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:18106]: > So we might want to lower that and substitute our vars in place of these?

Yeah that actually should be the case, dark reader should be capable of modifying that CSS rule.

**Comment by wan7teen-ping** [comment]
Hi I will help you with everything you need but you have to subscribe to
the youtube channel and I will fulfill all your requests

https://youtu.be/aIaqr4eq18g

Pada Sel, 4 Okt 2022, 4:25 AM Anton Bershanskiy ***@***.***>
menulis:

**Pull request merged**